precode.sh

```diff
@@ -7,37 +7,55 @@
 #   dir3
 #       dir4
 
+mkdir task task/dir1 task/dir2 task/dir3 task/dir3/dir4
```

Comment:
Можно использовать флаг `-p`
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```